app/views/users/new.html.erb

```diff
@@ -1,5 +1,18 @@
-<% provide :title, t(".user") %>
-<div class="center jumbotron">
-  <h1><%= t ".user" %></h1>
-  <p><%= t ".desc" %></p>
+<% provide :title, t(".sign_up") %>
+<h1 class="sign-up"><%= t ".sign_up" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user do |f| %>
```

Comment:
Sử dụng form_for, form_tag để làm chức năng này,
In parameter lên đây nhé.
So sánh giữa các loại này

Reply:
form_tag dùng để tạo form không có đối tượng
form_for dùng để tạo form có đối tượng

Reply:
![image](https://user-images.githubusercontent.com/68525190/89009410-b06f4b80-d336-11ea-8e11-6bded4d1ed5c.png)
form_for bên trên form_tag bên dưới ạ